src/api/settlements/settlements.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I think settlements should inside the merchant dir, where merchantConfig is because the parent of settlement in merchant